### DISCUSSION TITLE
Learning Liquidsoap: Is it possible to also generate an EPUB version?

### DISCUSSION THREAD

**Discussion by Moonbase59** [savonet/book#5] [Ideas]
I’m trying to learn more about Liquidsoap. It’s fine that we can have an A4 PDF, but reading that on an E-Reader (a Tolino Vision 5 in my case) is pure pain, especially if one’s eyesight isn’t that great anymore. With EPUB’s, one can much better adjust size & type.

So my question is: **Would you consider also generating the book as an EPUB please?**

Shouldn’t be difficult, looks like you use Markdown & Pandoc already. And would be a great help for all of us who keep their E-Reader next to the laptop when programming, for reference, and occasionally continue reading on a difficult part in their reading chair.

**Comment by smimram** [comment]
Just did this: http://www.liquidsoap.info/book/

Please tell me what it looks like, I don't have a reader at hand to test...

**Comment by Moonbase59** [comment]
Fan-tas-tic! Thank you so much, this is really helpful and works great: TOC, searches, display sizing, code wraparound, all good. I tested using

- Calibre’s Ebook Reader on Linux Mint 21.1
- Foliate 2.6.4 on Linux Mint 21.1
- Tolino Vision 5 E-Reader

Here are some screenshots:

![screenshot_20230627_220100](https://github.com/savonet/book/assets/3706922/f00d7093-e67a-42b8-a8e8-30028f9970f8)  
Tolino Landscape

![screenshot_20230627_220431](https://github.com/savonet/book/assets/3706922/df7f2c3c-efa5-4064-ad6f-8693593d1f89)  
Tolino Portrait, with settings menu

![screenshot_20230627_221402](https://github.com/savonet/book/assets/3706922/3a77539e-8af5-4523-a9b2-121081823d82)  
Tolino Portrait, with image

![The Liquidsoap book  EPUB  — E-book viewer_001](https://github.com/savonet/book/assets/3706922/f35bbf60-2e5d-4e21-a298-abb93c4216cb)  
Calibre E-book viewer on laptop

![The Liquidsoap book_001](https://github.com/savonet/book/assets/3706922/21bfcebf-5788-4fce-9a9d-b0b898371cdb)  
Foliate on laptop

**Reply by smimram** [reply]
Great! :)
